internal/sim/api_test.go

```diff
@@ -74,7 +74,7 @@ func (d *divModWorkload) Mod(ctx context.Context, x, y int) error {
 }
 
 func TestPassingSimulation(t *testing.T) {
-	s := New[divModWorkload](t, Options{})
+	s := New(t, &divModWorkload{}, Options{})
```

Comment:
Do you have a test here that tests the value receiver?

Reply:
Oops, added one.